### PR TITLE
Improve Documentation

### PR DESCRIPTION
**Pull request by JamesLaverack**
- Move usage documentation into the command help text from the README.
- Add some headline examples to the README.
- Document the `paranoia validate` config file structure more clearly.
- Add a simple exporter to generate manual pages, following the [Cobra documentation](https://umarcor.github.io/cobra/#documentation-generation).

### CODE REVIEW

File: cmd/options/outputs.go [hunk 1]
```diff
@@ -25,7 +25,26 @@ type Output struct {
 
 func RegisterOutputs(cmd *cobra.Command) *Output {
 	var opts Output
-	cmd.Flags().StringVarP(&opts.Mode, "output", "o", "pretty", "Output mode. Supported modes: "+strings.Join(outputModes, ", "))
+	cmd.Flags().StringVarP(&opts.Mode, "output", "o", "pretty", `
+The output mode controls how Paranoia displays the data, and what data is shown.
+Supported modes are *pretty*, *wide*, *json*, and *pem*.
```
Comment (ChaosInTheCRD) [inline cmd/options/outputs.go:30]: Will `StringVarP` render the emphasis (e.g. `*pretty*`)

File: cmd/root.go [hunk 2]
```diff
@@ -11,11 +11,52 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/manager/signals"
 )
 
-func newRoot(ctx context.Context) *cobra.Command {
+func NewRoot(ctx context.Context) *cobra.Command {
```
Comment (ChaosInTheCRD) [inline cmd/root.go:14]: Any reason why you're exporting this function? 🤔

File: hack/generate-manual/manual.go [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+	header := &doc.GenManHeader{
+		Section: "1",
+	}
+	const path = "man/"
+	_ = os.RemoveAll(path)
+	err := os.Mkdir(path, 0755)
+	if err != nil {
+		log.Fatal(err)
+	}
+	err = doc.GenManTree(cmd.NewRoot(context.Background()), header, path)
```
Comment (ChaosInTheCRD) [inline hack/generate-manual/manual.go:24]: Ah I see - sure 😄

**Review by ChaosInTheCRD — APPROVED** [review]

File: cmd/options/outputs.go (hunk 1 above)
Comment (JamesLaverack) [reply cmd/options/outputs.go:30]: In `--help`, no. In the man page, yes.
image
image

File: cmd/root.go (hunk 2 above)
Comment (JamesLaverack) [reply cmd/root.go:14]: The man page exporter needs it. Which is in a different package.

File: cmd/options/outputs.go (hunk 1 above)
Comment (ChaosInTheCRD) [reply cmd/options/outputs.go:36]: as long as you're happy with that 😄

File: cmd/root.go (hunk 2 above)
Comment (ChaosInTheCRD) [reply cmd/root.go:14]: yup

**Review by ChaosInTheCRD — APPROVED** [review]

**Pull request merged**